publish-swagger-docs.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+# Runs the api and backend db, grabs the generated swagger json spec and compares to what is in the
+# central reform-api-docs repo. Updates reform-api-docs spec if needed
+
+if [ "$TRAVIS_BRANCH" != "master" ]
+then
+  echo "This commit was made against the $TRAVIS_BRANCH and not the master! Not updating docs"
+  exit 0
+fi
+
+docker-compose up -d
+sleep 10
+REPO_NAME=$(echo ${TRAVIS_REPO_SLUG} | cut -f2- -d/)
+CURRENT_DOCS=$(curl https://hmcts.github.io/reform-api-docs/specs/${REPO_NAME}.json)
+NEW_DOCS=$(curl http://localhost:8800/v2/api-docs)
+docker-compose stop
+
+if [[ "$CURRENT_DOCS" != "$NEW_DOCS" ]]; then
+    echo "Update reform-api-docs"
+    mkdir swagger-staging
+    cd swagger-staging
+    git init
```

Comment:
would this achieve the same thing?:
```
git clone --depth 1 https://${GH_TOKEN}@github.com/hmcts/reform-api-docs.git -b master
```

Reply:
i'll have to try it - i had some grief doing a clone. I think because it's already in a git dir (draft-store) cloning within it and not doing an init made a bit mess :|

Reply:
strange, doing it in a subdirectory should be fine I would have thought